jbpm-workitems/jbpm-workitems-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -3,6 +3,9 @@
 
   <requiredProperties>
     <requiredProperty key="classPrefix"/>
+    <requiredProperty key="kieVersion">
+      <defaultValue>8.0.0-SNAPSHOT</defaultValue>
```

Comment:
can't we use project's version here that will get replaced on build time?

Reply:
@mswiderski sorry no the project version is going to be the users project version not the version of jbpm. so it might get replaced with 1.0.0 or whatever the user specifies. I can remove the default and force the user to enter the kie version if you want. let me know

Reply:
@tsurdilo BTW why is it called kieVersion? Won't it be copied to the resulting [pom.xml](https://github.com/kiegroup/jbpm/pull/966/files#diff-ef19c122b948a9674b48f07e7a21d616R12) as a value of version.org.kie property?